src/style.css

```
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 > .stars__star,
+.stars--1 :nth-child(n + 1),
+.stars--2 :nth-child(n + 2),
+.stars--3 :nth-child(n + 3),
+.stars--4 :nth-child(n + 4) {
+  background-image: url(images/star.svg);
+}
```

```suggestion
  background-position: center;
  background-image: url(images/star.svg);
}
```
just set default star to .starr_star element